manual_control.py

```diff
@@ -203,6 +205,8 @@ def main():
         help='window resolution (default: 1280x720)')
     args = argparser.parse_args()
 
+    args.host = '192.168.1.13'  # Change IP of the host server
```

Comment:
Remove this. It's convenient to hardcode your preferred IP but this will make it fail for others when running it without changes. If anything change the default IP in the argument parser.
```suggestion
```